website/siteConfig.js

```diff
@@ -17,28 +17,26 @@ const siteConfig = {
   projectName: 'USERDIVE',
   headerLinks: [
     { doc: 'getting-started', label: 'Usage' },
-    // {doc: 'doc4', label: 'API'},
-    // {page: 'help', label: 'Help'},
     { href: repoUrl, label: 'GitHub' },
     { blog: true, label: 'Blog' },
     { search: true }
   ],
   users,
-  headerIcon: 'img/logo_horizontal_white.png',
+  headerIcon: 'img/logo.svg',
```

Comment:
`logo_horizontal_white.png` include `USERDIVE` text, but the text rendere in Header with docusaurus v1.3.0 . 
Compare between local and [here](http://developers.userdive.org/)